### PR TITLE
Fixes SVGLoader arc parsing with zero radius path commands. See #20641

### PR DESCRIPTION
**Pull request by gregzanch**
Related issue: #20641

**Description**

If an SVG path has an arc command (A/a) with zero radius the path data gets thrown off. Adding a check in the path parser for zero radius fixes this:

```js
if( numbers[ j ] == 0 || numbers[ j + 1 ] == 0 ) {
  // draw a line if either of the radii == 0
  path.lineTo( point.x, point.y );
}
```

Also, to avoid unnecessary computation, a check for useless arc command was added:

```js
// skip command if start point == end point
if( numbers[ j + 5 ] == point.x && numbers[ j + 6 ] == point.y ) continue
```

### CODE REVIEW

File: examples/webgl_loader_svg.html [hunk 1]
```diff
@@ -104,8 +104,8 @@
 					"Defs": 'models/svg/tests/testDefs/Svg-defs.svg',
 					"Defs2": 'models/svg/tests/testDefs/Svg-defs2.svg',
 					"Defs3": 'models/svg/tests/testDefs/Wave-defs.svg',
-					"Defs4": 'models/svg/tests/testDefs/defs4.svg'
-
+					"Defs4": 'models/svg/tests/testDefs/defs4.svg',
+					"Zero Radius": 'models/svg/zero-radius.svg'
```
Comment (gregzanch) [inline examples/webgl_loader_svg.html:108]: This is just a test for the fix

File: examples/jsm/loaders/SVGLoader.js [hunk 2]
```diff
@@ -585,20 +595,28 @@ SVGLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 					case 'a':
 						var numbers = parseFloats( data );
-
+						console.log(parseFloats(data));
```
Comment (gregzanch) [inline examples/jsm/loaders/SVGLoader.js:598]: This has to get commented out

**Review by gregzanch — COMMENTED** [review]
Had to remove a console.log of the path data when debugging

File: build/three.js [hunk 3]
```diff
@@ -18327,7 +18327,7 @@
 					var bones = skeleton.bones;
 
 					if (capabilities.floatVertexTextures) {
-						if (skeleton.boneTexture === undefined) {
+						if (skeleton.boneTexture === null) {
```
Comment (Mugen87) [inline build/three.js:18330]: Please remove all build files from your PR.

**Comment by gregzanch** [comment]
I removed the build files in #20644

**Pull request closed without merging**